addons/sys_intercom/fnc_isInIntercom.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: ACRE2Team
+ * Checks if a unit is in the given intercom network of a vehicle.
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 1: Unit to be checked <OBJECT>
+ * 2: Intercom network <NUMBER>
+ *
+ * Return Value:
+ * Unit is in the given intercom network <BOOL>
+ *
+ * Example:
+ * [cursorTarget, acre_player, 0] call acre_sys_intercom_fnc_isInIntercom
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_unit", "_intercomNetwork"];
+
+private _unitsIntercom = _vehicle getVariable [QGVAR(unitsIntercom), []];
+
+if (_unitsIntercom isEqualTo []) then {
+    false
+} else {
+    _unit in ((_vehicle getVariable [QGVAR(unitsIntercom), []]) select _intercomNetwork);
+};
```

Comment:
It may be quicker to replace lines 22-28 with the following: 
`_unit in ((_vehicle getVariable [QGVAR(unitsIntercom), []]) param [_intercomNetwork,[]])`

It should be safer to do the following as it handles index out of bounds.